common/utils.go

```diff
@@ -75,12 +75,11 @@ func SetupLOG(logger *l.Logger, logLevel string) {
 	switch os.Getenv("INITSYSTEM") {
 	case "SYSTEMD":
 		fileFormat = l.MustStringFormatter(
-			`%{level:.4s} %{id:03x} %{shortfunc} > %{message}`)
+			`%{level:.4s} %{message}`)
```

Comment:
Go ahead and add it. Honestly, given the length of some of the functions this adds more noise than value for me. The function name without any package context has very limited value.